user-guide/modules/ROOT/pages/resources.adoc

```diff
@@ -41,7 +38,7 @@ Consider searching https://www.youtube.com/[YouTube] for online tutorials. There
 
 For more specific technical questions, Stack Overflow has many questions and answers:
 
-* https://stackoverflow.com/search?q=Boost+Library&s=d2d13545-4fbc-4f13-a029-08e8a70d2bdc[stack *overflow*: Boost Library]
+* https://stackoverflow.com/questions/tagged/boost[Stack Overflow: boost]
```

Comment:
Stack Overflow is not capitalized but boost now became lowercase.

Reply:
Stack Overflow should be capitalized - like Boost its logo is lower case, but otherwise caps should be used. Changed boost to Boost.

Reply:
Sorry... You're right. I meant

"Stack Overflow is now capitalized but boost now became lowercase."

Reply:
> Changed boost to Boost.

I still see "Stack Overflow: boost"